src/service/senderService.js

```diff
@@ -0,0 +1,64 @@
+import log from "loglevel";
+import sendMessage from "../post/send_message.js";
+import sendMessageChunks from "../post/send_message_chunks.js";
+import { checkConnection } from "./Helper.js";
+import payload from "../model/Payload.js";
```

Comment:
Importing lowercase payload does not look right, since class is not Payload. Don't think it is being used anyway.